test/e2e/node/runtimeclass.go

```diff
@@ -59,7 +59,9 @@ var _ = SIGDescribe("RuntimeClass", func() {
 		}
 		_, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.TODO(), pod, metav1.CreateOptions{})
 		framework.ExpectError(err, "should be forbidden")
```

Comment:
You can remove this line. `apierrors.IsForbidden` accepts nil and will return false, so the next `Failf` covers both cases.